frontend/src/css/url-builder.css

```
@@ -25,44 +25,41 @@ body#url-builder {
-    color: var(--dark-grey);
-
     &.with-error {
-        border: 2px solid var(--red);
         color: var(--red);
+        border: 2px solid var(--red);
     }
 
     a {
-        word-wrap: break-word;
         color: var(--blue);
+        word-wrap: break-word;
```

Stylelint (17.6.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)